### PR TITLE
Ie11/391 (1) fix buttons and clock

### PR DESCRIPTION
**Pull request by ruilealsilva**
78294002_2666865473398671_31558878525128704_n
75552979_2317255471922067_8455583569689968640_n

### CODE REVIEW

File: frontend/src/components/Header/StopClock.js [hunk 1]
```diff
@@ -90,10 +90,16 @@ class StopClock extends Component<Props, State> {
     const { currentTime } = this.state;
     const { startTime, countdown } = this.props;
     const percentage = getPercRemaining(currentTime, startTime, countdown);
+    const isIE = document.documentMode; // May be deprecated due to browser changes
```
Comment (bendiggle) [inline frontend/src/components/Header/StopClock.js]: Not a fan of this tbh. We should try and make it work for all browsers without the browser specific logic. Is there some sore of compromise that can be done?
If not, there's a module we can use to determine the browser type

Comment (ruilealsilva) [reply frontend/src/components/Header/StopClock.js]: I tried alot of different things to make it work on all the browsers at the same time. None of them were working properly in all at the same time. I can try it again but i think the end result will be the same. Should I use that module, then?

Comment (bendiggle) [reply frontend/src/components/Header/StopClock.js]: okie dokie if you're confident this is the best solution then let's do it. The module is https://www.npmjs.com/package/detect-browser

File: frontend/package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "carbon-components-react": "^6.6.3",
     "carbon-icons": "^7.0.7",
     "date-fns": "^1.29.0",
+    "detect-browser": "^4.8.0",
```
Comment (bendiggle) [inline frontend/package.json]: There's another package.json file you need to add this too. The level above frontend. It's for heroku

Comment (bendiggle) [reply frontend/package.json]: Once you've done that it's ready to go

**Pull request merged**